mch-questions.yml

```diff
@@ -45,9 +47,18 @@ questions:
     question: Have you been able to log into Minecraft on this computer before?
     choices:
     - text: 'Yes'
-      to: account-issues.unable-log-in.is-throttled
+      to: account-issues.unable-log-in.reset-password
     - text: 'No'
       to: technical-issues.ready
+  account-issues.unable-log-in.reset-password:
+    heading: ''
+    description: ''
+    question: Have you tried [resetting your password](https://account.mojang.com/password)? Make sure to check the spam folder. The reset password email may take up to 15 minutes to arrive.
+    choices:
+    - text: 'Yes'
```

Comment:
This doesn't seem like a good place for a "Yes" to me. Did you try -> several scenarios can occur here, which a "Yes" doesn't really work out well. "I changed my password" is probably more accurate.

Reply:
Changed it to Yes, and the new password doesnt work